node/src/command.rs

```diff
@@ -76,6 +76,7 @@ fn runtime(_id: &str) -> Runtime {
 }
 
 const KUSAMA_PARA_ID: u32 = 2119;
+const PASEO_PARA_ID: u32 = 2119;
```

Comment:
We can freely choose the para-id. Hence, I thought that we will pick the same as we have on Kusama.